### PR TITLE
Rename `ChampionMasteryManager.refreshAll` to ` ChampionMasteryManager.fetchAll`

### PR DESCRIPTION
**Pull request by MrBartusek**
`refreshAll` is a weird method name that doesn't match any other controller.

**It's a breaking change**. I have no idea how this repository handles breaking changes and deprections

### CODE REVIEW

**Comment by TheDrone7** [comment]
Breaking changes like this one are going to be held off until the next major release (which TBF is going to be sometime later this month).

File: docs/api/ChampionMasteryManager.md [hunk 1]
```diff
@@ -141,8 +158,6 @@ highest(n?: number): Promise<ChampionMastery>;
 
 #### .refreshAll ()
 
-Update the cache with the latest data for all champions' mastery data for this summoner.
-
```
Comment (MrBartusek) [inline docs/api/ChampionMasteryManager.md:162]: Also it seams like vuepress is not generating docs for deprecated properties

Comment (TheDrone7) [reply docs/api/ChampionMasteryManager.md:162]: The generator is actually manually written but yeah, it is designed to ignore private and deprecated properties when generating docs.

Comment (MrBartusek) [reply docs/api/ChampionMasteryManager.md:162]: privates should be ignored while deprecated  should be marked as such, probably with version when they got deprecated

**Comment by TheDrone7** [comment]
Sorry for the late reply, but please resolve the conflicts first.

**Comment by TheDrone7** [comment]
This still seems to contain the problems from #28 best to keep them separate for their own purposes.

**Comment by MrBartusek** [comment]
> This still seems to contain the problems from #28 best to keep them separate for their own purposes.

I guess I still don't know how to create branches 👀 I won't touch that till #28, 99% sure I will mess this up further

**Comment by TheDrone7** [comment]
This'll have to wait a long time. See the readme for the roadmap before V2.

**Comment by MrBartusek** [comment]
Oh yeah also, why are you making a task list in readme rather than issues 🤔 you can just create issues for these things and add them to a `2.0` milestone, it is much easier to organize work, discuss features and such

**Comment by TheDrone7** [comment]
Ideally I'd be using trello or something but rn I cba.

**Comment by MrBartusek** [comment]
If you enable them in repo settings you can actually use project boards, which is pretty much a trelo clone but with issues and pull requests integrations

**Comment by TheDrone7** [comment]
I've tried it before and honestly, it's a lot more complex than it has to be.

**Comment by TheDrone7** [comment]
Actually, since this is not a breaking change in the sense that it would break the existing codebases, it should be safe to merge this PR. Also, the docs have been updated to accommodate deprecated parts.

**Comment by TheDrone7** [comment]
Please resolve the conflicts and the tests can be rerun

**Review by TheDrone7 — APPROVED** [review]

**Pull request merged**